src/casp/services/deploy/cloudrun/README.rst

```diff
@@ -0,0 +1,34 @@
+Deployment Flavors
+==================
+
+This directory should contain various deployment configurations for the sizes of services used by the deployment GitHub action.
+These are JSON files named `[flavor].json` and the contents should look like:
+
+.. code-block:: json
+
+    {
+      "cas-admin": {
+        "cpu": "1000m",
```

Comment:
Probably a dumb question: what is a "1000m" CPU?

Reply:
Not dumb at all!  1000m CPU is the same as 1 CPU.  Often for shared resources, CPU is measured in sub-CPU units (e.g. milli-CPUs in this case)

Reply:
Oh that's really interesting!  I did not know that.  Thanks for explaining!